processor/elasticinframetricsprocessor/processor.go

```diff
@@ -60,13 +62,49 @@ func (p *ElasticinframetricsProcessor) processMetrics(_ context.Context, md pmet
 	for i := 0; i < md.ResourceMetrics().Len(); i++ {
 		resourceMetric := md.ResourceMetrics().At(i)
 		rm := resourceMetric.Resource()
-
 		for j := 0; j < resourceMetric.ScopeMetrics().Len(); j++ {
 			scopeMetric := resourceMetric.ScopeMetrics().At(j)
 			for _, r := range p.remappers {
 				r.Remap(scopeMetric, scopeMetric.Metrics(), rm)
 			}
 		}
 	}
+	// override=True will keep only the metrics that have been remapped based on the presense of OTelRemappedLabel label.
```

Comment:
What this comment describes is not an override action. Maybe that could be named to sth else like `drop_original` or something that clearly reflects the action here?

Reply:
I dont have any preferences. @rogercoll any ideas? TBH I kept the word override that is in the title of this issue. 
(I translated like override flag, overrides the default functionality)

Reply:
+1 on the `drop_original` naming, it looks more descriptive.  I just changed the issue's title with the "drop" keyword instead of "override" https://github.com/elastic/opentelemetry-lib/issues/97#issuecomment-2376625351  
Thanks!

Reply:
drop_original it is then ! I add it and docs updated

Reply:
Thank's!